File: dotnet/PoeLogsParser/Models/Entries/ChatMessageLogEntry.cs
using System;
using System.Collections.Generic;
using PoeLogsParser.Enums;

namespace PoeLogsParser.Models
{
    public class ChatMessageLogEntry : LogEntry
    {
        public string Player { get; set; }
        public string Message { get; set; }
        
        public ChatMessageLogEntry(LogEntry entry) : base(entry.Raw, entry.Time, entry.Types, entry.Tag)
        {
        }
    }
}

File: js/src/models/entries/ChatMessageLogEntry.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.ChatMessageLogEntry = void 0;
const LogEntry_1 = require("./LogEntry");
class ChatMessageLogEntry extends LogEntry_1.LogEntry {
    constructor(entry) {
        super(entry.raw, entry.time, entry.types, entry.tag);
    }
}
exports.ChatMessageLogEntry = ChatMessageLogEntry;
